### ISSUE TITLE
"App delay" is too annoying to continue using Franz

### ISSUE THREAD

**Issue by ElectricWarr** [meetfranz/franz#1167]
The "app delay" feature after upgrading to Franz 5.0.0-beta (love the new UI!) is so annoying it's driven me back to using a few Chrome tabs and the odd native app.

If the delay only appeared on quitting/restarting Franz then I could deal with it, and after some time I might consider subscribing - but being forced to wait after only a few minutes with it in the background is unacceptably obstructive.

I would have considered a subscription if the app didn't hold my messages to ransom almost every single time I check.

This might not be the right place to point this out, but I couldn't see a better one.

**Comment by adlk** [comment]
Thank you for your feedback.

"after a few minutes" in that case means after 60 minutes when the app gets focus again. The current timer is however subject to optimization.

**Comment by ElectricWarr** [comment]
FWIW I'd suggest "on every startup, and once per day if the app is left running" would be a reasonable compromise.

You'd get your message through without driving away your users.

**Comment by Gaboris** [comment]
What does this "App delay" actually mean?
I've been using Franz since v4 and haven't noticed any delays especially not several times a day. Is this service specific or just something in the background that I haven't payed attention to yet?

**Comment by vpont** [comment]
I just started testing Franz a couple of days ago and I have to agree with @ElectricWarr.
Please consider the option "on every startup" already mentioned.
Thanks and great work!

**Comment by psharpep** [comment]
@adlk I'm not sure if it's still worth providing input on this, but I wanted to second @ElectricWarr 's opinion that the app delay is too annoying. I've actually gone back to v4 (despite the many wonderful UI improvements in v5) because this breaks up my workflow too much.

I think the "show message on every startup" option would be a good middle ground. (Alternatively, consider making the supporter license a pay-what-you-want plan for personal users: $40/year is too much to justify on my student budget, although I would gladly pay $20/year.) Perhaps another option would be to keep the frequency of the app delays, but reduce their duration to something like 3 seconds.

Thank you for your app, and I hope you'll consider something along these lines.

**Comment by milind-shakya-sp** [comment]
Pretty annoying tbh. I just installed the app today, and am already presented with this delay like 5 times already. Uninstalling it for now.

**Comment by marcolopes** [comment]
I tested FRANZ yesterday, and been using it for 2 days, but will give up and return to NATIVE apps because of the DELAY. I know the author needs support, but this is nonsense... i want an app to be as FAST and responsive as i can... being DELAYED several times a day is NOT acceptable. Will not recommend FRANZ to anyone else (it was recommended to me, and my friends DON'T have the DELAY!!! why? Older users?)

**Comment by ronkinoz** [comment]
I agree with others. Have had the delay 4 times this afternoon. It's bordering on driving me to delete Franz. The delay outweighs the benefits of Franz 5.

**Comment by junalmeida** [comment]
I agree, I am considering alternatives.

**Comment by marcolopes** [comment]
How about a FORK?

**Comment by marvin-w** [comment]
I also agree with the others.

**Comment by nirsky** [comment]
Have to agree as well, just joined but this is too annoying, too bad...
Guess I'd have to go back to the native apps (or fork it...).

**Comment by MitchTalmadge** [comment]
I just downloaded Franz, was excited to use it, and then saw this delay. Honestly, this is disgusting. You say this is made with "lots of love" but nagging your users is not very loving. I am seriously considering uninstalling Franz and finding an alternative. I would be perfectly fine with a delay on startup, and it may even convince me to pay one day, but holding my messages ransom is wrong and will do nothing but piss your users off, as evident in this issue.

You could also offer a lifetime license rather than a subscription model. I'd purchase a lifetime license to get this to go away.

**Comment by igreil** [comment]
I urge you to consider a one-time payment option for non-commercial users. There is no way for me to justify a monthly payment for this program. Software as a service doesn't work for me, I prefer to make outright payments if and when I have the budget for it.

That nag screen? Pretty silly in open source software. Still mildly annoing, though.

**Comment by xthursdayx** [comment]
Yeah, I just wanted to add that I love this software and have been using it (and spreading the word) for years now. However, the price is far too much for my student budget and the whole nag screen + walled off sections of the software seem antithetical to the idea of this as open-source software, so, unfortunately, I am jumping ship.

**Comment by Ragetip** [comment]
I second that. Franz is a good program, and whoever made this has all the right to ask for compensation for his efforts.
Thing is, the nag screen is very aggressive. Even old 3.11 Windows programs didn't do this that much often. I'm sure that with funding, Franz can become even greater than before, but it also needs to work well with his concept of flexibility, something that i'm sure most users had installed Franz just for that. 
Having a nag screen for every startup or once every 24Hrs would be better, or even yet a one time payment would be great. I'd surely buy this program, not subscribe to it since it uses other, free versions of already available services. I could just put a browser session for all the web versions of my favourite messengers and i would be done.

TL;DR please don't ruin Franz with this payment plan in mind. You earned your merit to be payed, but not like this. Thank you.

**Comment by peterdsharpe** [comment]
And the open source community delivers again!

No more paywall:
https://github.com/kytwb/franz

**Comment by SYip** [comment]
This "feature" is nonsense for an open source application as anyone can fork it and remove the "feature". It will drive people out of your main repository.

Rather than introducing annoying "feature", it would be much better to introduce enterprise features. i.e. Gitlab

**Comment by sinedied** [comment]
An alternative to Franz without the nag screen: https://getstation.com

**Comment by MitchTalmadge** [comment]
I tried station but it doesn't have a tray icon so if you close the app, it's gone. It's on their todo list but really should have been included in version 1.

**Comment by marcolopes** [comment]
Tried STATION too, but it's TOO SLOW (opening the apps one at a time and very slow...), hogs 50% of my i5 cpu power, and closing the window does not close the APP (it still runs with 50% CPU) but no tray icon.

**Comment by tamm** [comment]
Hey, I just found this because I got a friend to download recently and then I got this:

"Franz thinks I like it enough to pay for it, and forces me to wait 10 seconds every now and then."
"I'm a developer, I'm used to compiling code!"

I'm not sure I can really spread the word any more, I don't want to make my friends wait every hour (Time listed above). I had no idea despite being a daily user since several years so I guess I just don't get these for some reason?

**Comment by Ragetip** [comment]
> Hey, I just found this because I got a friend to download recently and then I got this:
> 
> "Franz thinks I like it enough to pay for it, and forces me to wait 10 seconds every now and then."
> "I'm a developer, I'm used to compiling code!"
> 
> I'm not sure I can really spread the word any more, I don't want to make my friends wait every hour (Time listed above). I had no idea despite being a daily user since several years so I guess I just don't get these for some reason?

It's the version 5 that has this. The old version, wich doesn't automatically upgrade to 5, do not have those nag screen. I had to upgrade to 5 (you have to do that manually) because Skype wasn't working anymore (not that it really works in Franz, it never shows notifications). 
So if you have an old Franz version that still has some compatibility with some services, all good. No nags. Just old options.
But the version 5 onwards had, and still has, the nag screen.

**Comment by Gaboris** [comment]
I haven't experienced this App delay so far and I've been using Franz since 4.0 and updated to 5.0 right away. Maybe upgrading Franz instead of making a clean install of it broke this functionality and we just got lucky?

Could someone explain what this is exactly?
Does it actually block your access to your services every hour or so for a few seconds by covering the screen with a "promo message" or something?

**Comment by Ragetip** [comment]
It essentially looked like this:

![image](https://user-images.githubusercontent.com/48887363/55966636-08cd9600-5c79-11e9-8353-8d1a7014cd2b.png)

Yes, it stopped all services until the countdown reached 0.
To be perfectly honest, right now it is doing it **when i just open the program**. It is not doing it while working, i might've seen the nag screen for a split second. Maybe a recent update? I don't see anything in the changelogs. If this happens during the day i'll post about it.
If it's an update that passed the nag screen on launch only, i'm 100% fine by it.

**Comment by Gaboris** [comment]
Thank you!
Got it now. Hope it actually got patched already and won't be as aggressive in the future.

**Comment by edwin-jones** [comment]
I've had to uninstall franz as this was just too annoying. There must be better ways to monetize. Prompt once a day maybe?

It's a shame as I really like the app.

**Comment by jpwilbur** [comment]
Definitely disappointed in the frequency of this "ransom message". I would have been happy to pay for a one-time purchase, but when there are so many alternatives (I'm installing Station as we speak https://getstation.com/), it doesn't make sense to put this on a recurring payment plan.

And, with almost no development on this for so long, and a huge line of bugs and issues to fix, I have to doubt it will continue to be developed and maintained long-term. You had a great thing going for you @adlk . Sad to leave it now that it's structured as a for-profit project.

**Comment by BrokedTV** [comment]
Yes, sorry but same, just moved to station, i really get that development need support, but this is too much even for testing.  Popping out 2/4 times a day could  be ok, maybe in a smart way depending on usage ( when on low load, not many notification), but right now it's too much, it break my work flow too often.

**Comment by sinedied** [comment]
Also moved to station by now, still sad that Franz made this move. I would understand and even support a one-time support payment, but a subscription for that kind of app that really not fits well, especially for individual usage.

**Comment by danielroe** [comment]
I've also now moved to [Station](https://getstation.com/) due to this annoying nag, though I will keep an eye on [Ferdi](https://github.com/kytwb/ferdi).

This is not the way to monetise an open-source project.

**Comment by rantwijk** [comment]
I installed Franz this morning, I'm already considering uninstalling it because of this. Thanks for the tip about Station.

**Comment by hanrok** [comment]
It's so annoying! I install it today as an alternative to Station because it has little bit lags, but this feature is severe more than any other bug so I might leave it and move back to Chrome.

**Comment by bittner** [comment]
> An alternative to Franz without the nag screen: https://getstation.com

I looked at it. The features are convincing ("Auto Sleep" to save CPU and RAM, wow!). Unfortunately, it doesn't seem to be Free/Libre open source software. Everyone who cares for freedom should stay away from it.

Look at Franz, for example: Forking it in the name of freedom is only possible because the source code is available! 

## References

- [getstation/station-canary](https://github.com/getstation/station-canary) -- _no code, just binaries - classic closed source model_

**Comment by bittner** [comment]
[Rambox](https://rambox.pro/) is another alternative, though personally I don't like its layout and the missing polish.

**Comment by NickPiscitelli** [comment]
This is not a way to monetize a platform. I will be finding alternatives.

**Comment by julian916** [comment]
Thank you for the ones that recommend [https://getstation.com/] I just moved to that.
Bye bye franz. This is not the way to monetize a platform.

**Comment by bittner** [comment]
Please, stop promoting Station! Seriously! It's creepy, non-free software!

**Comment by julian916** [comment]
Station is a free desktop application.

**Comment by bittner** [comment]
Show me the source code and I tell you whether it's free. If anything it's [freemium](https://en.wikipedia.org/wiki/Freemium). You have no code, you have no control. Use it if you don't care, but don't promote it.

I [explained things above](https://github.com/meetfranz/franz/issues/1167#issuecomment-495997466) a little bit more in detail, I believe.

**Comment by julian916** [comment]
Free software and open source are 2 different things. I need an app that dont make me wait for use it. 
You dont need to see the code to use an application for free 🤷‍♂ 
Nobody cares about your conspiracies ideas just need a free app to integrate multiple services without  an annoying timer in it.

**Comment by BrokedTV** [comment]
Open source is always to be preferred but not when there are such feature that limit the usability, it's a poor choice.

**Comment by kytwb** [comment]
I started [kytwb/ferdi](https://github.com/kytwb/ferdi) as a personal hard fork as I was also being bugged by the app delay. Setup'd Travis/Appveyor for continuous delivery of the assets and uploaded them in the latest release.

**Comment by billnguyensg** [comment]
As many ppl have pointed out, this is not the way to monetize.

**Comment by ZeroCool940711** [comment]
The stupid delay thing makes me want to stop using Franz, what a shame as it is a really nice app.

**Comment by gynekolog** [comment]
#metoo

**Comment by nasmithan** [comment]
Also, who on earth wants to start paying a monthly subscription for software like this that has free and very comparable alternatives? I was ready to pay if it was $20 or maybe even $25. But €4/month?! No thank you.

**Comment by waclock** [comment]
Started using Station a couple of weeks ago. Haven't looked back, but honestly I'm kind of sad that Franz "forced" me to stop using it. It's an amazing piece of software and I really hope you can find a proper way of monetizing it.

**Comment by billnguyense** [comment]
Switched to Station for almost 2 months. There are some issues in Station too, for example, links are not clickable in Skype... but I am fine with it. Still better than the delay in Franz.

**Comment by Ragetip** [comment]
> Switched to Station for almost 2 months. There are some issues in Station too, for example, links are not clickable in Skype... but I am fine with it. Still better than the delay in Franz.

Neither are in my Franz. I click skype links and they won't open up, unless i copy and paste them outside. Before an update, i could click them, only to find out an error page. Maybe Microsoft doesn't quite like to be filtered out like that.

But yeah, i'm still using Franz. It can scream all it wants, i won't pay for it :P lately it's offering a "14 day free trial". Yup sure. Until it has a lifetime license it's a no-go, they can even extend that timer to 2,5, 10 minutes of delay, i won't pay a cent if it's a subscription only, for that kind of service!

**Comment by sshipway** [comment]
My copy of franz just upgraded itself and this is going on.  I'm getting rid of it now, nagging delay screens are not worth the slight added convenience.

**Comment by eduardo-bueno-ifood** [comment]
I literally can't think of a worst way to nag for support than this. I'm fine having ADs in the app for example, but this is some next level thing. Congrats to whoever came up with the idea.

**Comment by sshipway** [comment]
@eduardo-bueno-ifood I suggest you take a look at Rambox. that's what I replaced it with.  No nag sdelay and a better UI

**Comment by rantwijk** [comment]
I've since moved to Ferdi, a community fork of Franz with additional features and no nag screen: https://github.com/getferdi/ferdi

**Comment by TheGrave** [comment]
> I've since moved to Ferdi, a community fork of Franz with additional features and no nag screen: https://github.com/getferdi/ferdi

Thank god for that, I'm off that stupid nagging Franz shit once and forever! RIP suckers!

**Comment by strydyr** [comment]
I just found out about this app yesterday, on a recommendation.

I loved it.  Until the first nag.  I'd been considering how to use the "3 services", and I'm sure an upgrade would've eventually happened, even if I prefer one-time payments, and "owning" software.

I'm here because I was looking to see if the nag was intentional.  Surely, nobody would do this on purpose, and it had to be another issue triggering the nag so often....

Sadly, I'll be uninstalling Franz in a few minutes.  I'm only commenting here because I hope the developers will understand that this model will not win people over.  Franz is truly close to greatness, and with additions like a "combined inbox" service, it has the potential to replace many applications that are part of my workflow, and rescue me from browser-tab hell.

I'm sorry it didn't work out.  It's not me, though, it's you.  I sincerely thank you for not making the nag a "bomb" that I'd find out about in 30 days.  It's best that I pull the band-aid off quickly, before the adhesive sets in.

**Comment by Ragetip** [comment]
> I just found out about this app yesterday, on a recommendation.
> 
> I loved it. Until the first nag. I'd been considering how to use the "3 services", and I'm sure an upgrade would've eventually happened, even if I prefer one-time payments, and "owning" software.
> 
> I'm here because I was looking to see if the nag was intentional. Surely, nobody would do this on purpose, and it had to be another issue triggering the nag so often....
> 
> Sadly, I'll be uninstalling Franz in a few minutes. I'm only commenting here because I hope the developers will understand that this model will not win people over. Franz is truly close to greatness, and with additions like a "combined inbox" service, it has the potential to replace many applications that are part of my workflow, and rescue me from browser-tab hell.
> 
> I'm sorry it didn't work out. It's not me, though, it's you. I sincerely thank you for not making the nag a "bomb" that I'd find out about in 30 days. It's best that I pull the band-aid off quickly, before the adhesive sets in.

I've finally removed that program after convincing some people to jump from a service to the other (I had to watch over 3 services). I now have a shortcut that opens 2 messengers sessions at once. Believe me: it's cheaper! : P
My Franz had a bug where the nag screen would come off if i clicked the app icon two times. I worked around that for the longest time, but some perfomance issues plus some of my concerns about privacy convinced me to jump on the old fashioned - but optimized - wagon.
As for now i'm convinced that Franz wants to be onboard middle to big businesses. Us random users or little business owners are not the target anymore. Plus, the subscription model is something that is much more popular (and hated) for any kind of program or service these days.
Eeh, what can you do.

**Comment by djly** [comment]
Created an account yesterday, setup 3 services. Today I'll be uninstalling because of this delay feature. Adding artificial inconvenience to get people to sub. I'm not sure I can get behind this company if that's their monetization strategy
